pkg/apis/azureprovider/v1alpha1/azuremachineproviderstatus_types.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
2019 since this is a new file
```suggestion
Copyright 2019 The Kubernetes Authors.
```